### DISCUSSION TITLE
404 when clicking on fleet queries/packs

### DISCUSSION THREAD

**Discussion by gr4ssi** [Security-Onion-Solutions/securityonion#6260] [Unsupported Versions] [answered]
Hi, just playing around with Fleet on latest SO 2.3.80.
So I'm in Fleet / queries or Fleet/ packs. They link to each other on the right hand side. There, the links don't work, because a "/fleet/" is missing between the IP and the page I'm are trying to open.
So the link would be https://192.168.11.17/packs/1 where it actually should be https://192.168.11.17/fleet/packs/1
![image](https://user-images.githubusercontent.com/19840345/142218414-9f32c4e3-0f63-4fd0-818b-dae6a26c4f4a.png)

This happens, when you are in Queries and want to select an accompanying pack and vice versa.

**Comment by defensivedepth** [comment, marked as answer]
Thanks for reporting this! It is actually a bug in FleetDM, I have created a new issue over in their repo: https://github.com/fleetdm/fleet/issues/3039
